distributed/client.py

```diff
@@ -117,9 +117,6 @@
     "pubsub": PubSubClientExtension,
 }
 
-# Placeholder used in the get_dataset function(s)
-NO_DEFAULT_PLACEHOLDER = "_no_default_"
```

Comment:
Driveby: Replaced `NO_DEFAULT_PLACEHOLDER` with existing and already imported `no_default` from `distributed.utils`